modules/mco-update-boot-images-configuring.adoc

```diff
@@ -7,55 +7,60 @@
 
-:FeatureName: Boot image management on {vmw-short}
-include::snippets/technology-preview.adoc[] 
+[NOTE]
+====
+Because the boot image management feature for worker nodes is default for the {gcp-short} and {aws-short} platforms, the `managedBootImages` configuration does not appear in the machine configuration object. To enable the feature for control plane machine sets without disabling the feature for worker machine sets, you must expressly add the configuration for both the control plane and worker machine sets, as shown in the following procedure. If you add only the configuration for control plane machine sets, due to default behavior, the Machine Config Operator (MCO) overwrites the configuration for the worker machine sets. 
+====
 
-Enabling the feature updates the boot image to the current {product-title} version. If the cluster is again updated to a new {product-title} version in the future, the boot image is updated again. New nodes created after enabling the feature use the updated boot image. This feature has no effect on existing nodes.
+Enabling the feature updates the boot image to the {op-system-first} boot image version appropriate for your cluster. If the cluster is again updated to a new {product-title} version in the future, the boot image is updated again. New nodes created after enabling the feature use the updated boot image. This feature has no effect on existing nodes.
 
 .Prerequisites
 
-* For {vmw-short}, enable the `TechPreviewNoUpgrade` feature set on the cluster. For more information, see "Enabling features using feature gates".
+* If you are enabling boot image management for control plane machine sets, you enabled the required Technology Preview features for your cluster by editing the `FeatureGate` CR named `cluster`:
 +
-[NOTE]
-====
-Enabling the `TechPreviewNoUpgrade` feature set cannot be undone and prevents minor version updates. These feature sets are not recommended on production clusters.
-====
-+
-Wait until the `managedBootImagesStatus` stanza displays in the `MachineConfiguration` object. 
+[source,terminal]
+----
+$ oc edit featuregate cluster
+----
 +
+.Example `FeatureGate` CR
 [source,yaml]
 ----
-apiVersion: operator.openshift.io/v1
-kind: MachineConfiguration
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
 metadata:
   name: cluster
-# ...
-status:
-# ...
-  managedBootImagesStatus:
-    machineManagers:
-    - apiGroup: machine.openshift.io
-      resource: machinesets
-      selection:
-        mode: None
+spec:
+  featureSet: TechPreviewNoUpgrade <1>
 ----
+<1> Enables the required `ManagedBootImagesCPMS` feature gate.
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. This feature set allows you to enable these Technology Preview features on test clusters, where you can fully test them. Do not enable this feature set on production clusters.
```

Comment:
Consider the instruction up front
```suggestion
Do not enable this feature set on production clusters. Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. This feature set allows you to enable these Technology Preview features on test clusters, where you can fully test them.
```

Reply:
This is wording that is used elsewhere in the docs in similar ways. Wonder if it is worth updating throughout?